WalletWasabi/Services/WalletService.cs

```diff
@@ -1279,6 +1290,18 @@ public Operation(Script script, Money amount, string label)
 				TxOut output = tx.Outputs[i];
 				var anonset = (tx.GetAnonymitySet(i) + spentCoins.Min(x => x.AnonymitySet)) - 1; // Minus 1, because count own only once.
 				var foundKey = KeyManager.GetKeys(KeyState.Clean).FirstOrDefault(x => output.ScriptPubKey == x.P2wpkhScript);
+
+				// If we have address reuse, give all of them anonset 1
+				var coinsWithSameKey = Coins.Where(x => x.HdPubKey == foundKey).Distinct().ToList();
+				if (coinsWithSameKey.Any())
+				{
+					anonset = 1;
+					foreach (var c in coinsWithSameKey)
```

Comment:
maybe use `coin` instead of `c` as a variable.

Reply:
I would but there is a coin variable in this scope already